Containerd/self-assessment.md

```diff
@@ -0,0 +1,333 @@
+# Self-assessment
+The Self-assessment is the initial document for projects to begin thinking about the
+security of the project, determining gaps in their security, and preparing any security
+documentation for their users. This document is ideal for projects currently in the
+CNCF **sandbox** as well as projects that are looking to receive a joint assessment and
+currently in CNCF **incubation**.
+
+# Self-assessment outline
```

Comment:
This is superflous, see [this example](https://github.com/cncf/tag-security/pull/1181/files) for what to remove

```suggestion
```